frontend/app/controllers/Event.scala

```diff
@@ -85,13 +87,24 @@ trait Event extends Controller {
     ))
   }
 
+  private def chronologicalSort(events: Seq[model.RichEvent.RichEvent]) = {
+    events.sortWith(_.event.start < _.event.start)
+  }
+
   def list = CachedAction { implicit request =>
     val pageInfo = PageInfo(
       CopyConfig.copyTitleEvents,
       request.path,
       Some(CopyConfig.copyDescriptionEvents)
     )
-    Ok(views.html.event.guardianLive(guLiveEvents.getEventPortfolio, pageInfo))
+    Ok(views.html.event.guardianLive(
+      EventPortfolio(
+        guLiveEvents.getFeaturedEvents,
+        chronologicalSort(guLiveEvents.getEvents ++ discoverEvents.getEvents),
+        (guLiveEvents.getEventsArchive ++ discoverEvents.getEventsArchive).headOption.map(chronologicalSort(_).reverse),
```

Comment:
Might be worth pulling this out as a `val` as it's doing quite a lot and the `reverse` it hidden from this view.